CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- an image in the singleview will be displayed in an image-tag instaed as a background image
```

Comment:
I propose this wording to keep this shorter:

Use an `img` instead of background image for the image in the single view

Reply:
Also, please add the link to the PR ID.